daemon/cmd/daemon_main.go

```diff
@@ -490,6 +490,14 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	option.BindEnv(vp, option.K8sServiceCacheSize)
 	flags.MarkHidden(option.K8sServiceCacheSize)
 
+	flags.Int(option.K8sServiceDebounceBufferSize, 128, "Number of distincts services to buffer for event debouncing")
```

Comment:
I would argue that it makes sense to enable it by default to make sure that it is on for the environments that really need this. Otherwise we'll be in a situation where the only way to get is enabled is by having a user of Cilium open a github issue or similar complaining about how badly Cilium behaves.

We could drop the default delay to e.g. 200ms as that's probably already enough to reap the benefits.

Reply:
Sounds good to me